### ISSUE TITLE
Configuration of merge.py by config.h

### ISSUE THREAD

**Issue by aeickho** [travisgoodspeed/md380tools#169]
I wish me...
   a solution to enable / disable hooks by config.h

close this issue when this solution is included, or my idea make no sense.

**Comment by travisgoodspeed** [comment]
Yes, we do need some sort of configuration manager, like `make menuconfig` on Linux.

**Comment by travisgoodspeed** [comment]
Some of the #181 patches get us closer to this, with explicit configuration options for each of the features.  I'll be sure to support config.h in the new patcher, which is coming up soon on my TODO list.
